docs/architecture.md

```diff
@@ -30,7 +28,7 @@ Note that while all traces are generated, only few are sampled. Sampling a trace
 By default, Jaeger client samples 0.1% of traces (1 in 1000), and has the ability to retrieve sampling strategies from the agent. 
 
 ![Context propagation explained](images/context-prop.png)
-*Context propagation*
+*Illustration of context propagation*
```

Comment:
I can go with no captions and simply refer to the images as "in the figure above/below". If we want captions they need to be styled properly, right now looked just weird & out of place.